spec/protocol/attributes.adoc

```diff
@@ -32,3 +42,19 @@ A CBOR tag on the payload value (not in the bytestring) must be added by those p
 The tags are specified per attribute.
 
 A tag MUST NOT be added to the result if the value follows the schema specified by the attribute specification.
+
+== Request Attributes
+TBD
+
+== Response Attributes
+TBD
+
+== Unknown Attributes
+
+A server who receives a request with an attribute it does not recognize MUST automatically return an error with code `UnknownRequestAttribute` (-12) and a list of attributes that caused the error.
+No further action should be taken; an unknown attribute should have no side effect.
+The client is then welcome to resend the request without the faulty attribute.
+
+A client receiving a response with an attribute it does not recognize should ignore the attribute, and likely report a warning.
+In some cases the payload might change, so a client that sees unknown attributes in a response should try its best to interpret its payload.
+As an example, a CLI tool that sees `[1, ...]` attribute and doesn't support asynchronous responses should error out with a comment that the payload was empty (see xref:../../attributes/response/1_async.adoc[Response Attribute 1 - Async] for more details).
```

Comment:
Because you're in a situation where you sent a request, the request executed an action, you received a valid response, but there are attributes in the response you do not understand. An error might imply the user should re-execute the message. But the message was executed. You're having an error _after the fact_. The message would be like "Error: Success".

Reply:
Make sense. It wasn't clear in the spec; I would describe it like you just did.

Reply:
Added more text.